### PR TITLE
Feature/hedgedoc logo

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Navbar, front page logo and loading screen

### Description
This PR replaces the old codimd icon with the new hedgedoc logo.

### Steps

- [x] added implementation
- [ ] added / updated tests
- [ ] added / updated documentation
- [ ] extended changelog

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
-    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#333333">
+    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#9b1b01">
     <link rel="shortcut icon" href="/icons/favicon.ico">
-    <meta name="msapplication-TileColor" content="#333333">
+    <meta name="apple-mobile-web-app-title" content="HedgeDoc">
+    <meta name="application-name" content="HedgeDoc">
+    <meta name="msapplication-TileColor" content="#e8a85a">
     <meta name="msapplication-config" content="/icons/browserconfig.xml">
-    <meta name="theme-color" content="#ffffff">
+    <meta name="theme-color" content="#e8a85a">
```
Comment (ErikMichelson) [inline public/index.html:16]: This color is not part of the official theme colors.

File: public/index.html [hunk 2]
```diff
@@ -7,11 +7,13 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/icons/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/icons/favicon-16x16.png">
     <link rel="manifest" href="/icons/site.webmanifest">
-    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#333333">
+    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#9b1b01">
```
Comment (ErikMichelson) [inline public/index.html:10]: This color is not part of the official theme colors.

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
There's an unnecessary empty html file included in this PR. Could you remove it?

File: public/icons/site.webmanifest [hunk 3]
```diff
@@ -13,7 +13,6 @@
             "type": "image/png"
         }
     ],
-    "theme_color": "#ffffff",
-    "background_color": "#ffffff",
-    "display": "standalone"
+    "theme_color": "#303030",
```
[the remark this answers is not in the document]
Comment (mrdrogdrog) [reply public/icons/site.webmanifest]: done

File: public/index.html (hunk 1 above)
Comment (mrdrogdrog) [reply public/index.html]: Yes, that's right. I forgot to change the index.html.

File: public/index.html (hunk 2 above)
Comment (mrdrogdrog) [reply public/index.html]: done

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Pull request merged**